queries.sql

```diff
@@ -8,3 +8,69 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT * from Animal22 WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Transactions
+begin;
+update animals
+set species = 'unspecified';
+select species from animals;
+rollback;
+select species from animals;
+
+begin;
+update animals
+set species = 'digimon'
+where name like '%mon';
+
+begin;
+update animals
+set species = 'pokemon'
+where name not like '%mon';
+
```

Comment:
- [ ] Kindly commit this transaction. 👍 You have begin this transaction but didn't commit and because of that reason your data will not change by this transaction 🤔